src/component-queries/Diseases.tsx

```diff
@@ -36,8 +36,8 @@ const DiseaseTemplate = (props: QueryResult) => {
             disease.frontmatter;
             return {
                 name,
-                geneSymbol: gene[0].frontmatter.symbol,
-                geneName: gene[0].frontmatter.name,
+                geneSymbol: gene?.[0]?.frontmatter.symbol,
```

Comment:
a disease should always have a gene, why was this change needed?

Reply:
thank you for catching that! I was fixing around type errors and missed to revert this one back.